__tests__/app.test.js

```diff
@@ -109,3 +107,38 @@ describe('GET /api/articles', () =>{
         });
     })
 })
+
+describe('GET /api/articles/:article_id/comments', () =>{
+    test('GET 200 returns an array of comments from an article ID sorted in descending order', () =>{
+        return request(app)
+        .get('/api/articles/5/comments')
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.article;
+          comments.forEach(comment => {
+            expect(comment).toHaveProperty('comment_id');
+            expect(comment).toHaveProperty('votes');
+            expect(comment).toHaveProperty('created_at');
+            expect(comment).toHaveProperty('author');
+            expect(comment).toHaveProperty('body');
+            expect(comment).toHaveProperty('article_id');
+          });
+        });
+    })
+    test('returns 400 for invalid sort_by parameter', () => {
```

Comment:
We didn't need to include queries at this stage for this task but if you have it's important to ensure it's a fully tested implementation, we only have an error test for a query attached to this endpoint, no happy path. Consider if its best to leave this out for now or to finish it.